client/app/components/message-list.coffee

```diff
@@ -200,16 +200,18 @@ MessageList = React.createClass
                         # refresh
                         if not @state.edited
                             div className: btnClasses,
-                            refreshClasses = 'fa fa-refresh'
-                            if @props.refreshes.length > 0
-                                refreshClasses += ' fa-spin'
-                            div className: 'btn-group btn-group-sm message-list-option',
-                                button
-                                    className: 'btn btn-default',
-                                    type: 'button',
-                                    disabled: null,
-                                    onClick: @refresh,
-                                        span className: refreshClasses
+                                if @props.refreshes.length is 0
+                                    button
+                                        className: 'btn btn-default',
+                                        type: 'button',
+                                        disabled: null,
+                                        onClick: @refresh,
+                                            span className: 'fa fa-refresh'
+                                else
+                                    img
+                                        src: 'images/spinner.svg'
+                                        alt: 'spinner'
+                                        role: 'spinner'
```

Comment:
As `[role=spinner]` doesn't exist in ARIA spec, I think it won't bring anything here and can make things more confuse.

If we cannot handle the throbbers in pseudo-elements, then prefer a class here to target it with CSS. Even if custom roles are allowed by the specs, a throbber should not be in the markup as it's only a state indicator, so a class doesn't hurt me as well :wink:.